crates/fluvio-benchmark/benches/producer_consumer.rs

```diff
@@ -0,0 +1,82 @@
+use criterion::*;
+use fluvio_benchmark::{
+    setup,
+    bench_env::{
+        FLUVIO_BENCH_RECORD_NUM_BYTES, EnvOrDefault, FLUVIO_BENCH_RECORDS_PER_BATCH,
+        FLUVIO_BENCH_SAMPLE_SIZE, FLUVIO_BENCH_MAX_BYTES_PER_BATCH,
+    },
+    benches::{run_throughput_test, run_overhead_test, run_throughput_no_check},
+};
+
+use criterion::async_executor::AsyncStdExecutor;
+
+fn bench_throughput(c: &mut Criterion) {
+    println!("Test configuration:");
+    println!(
+        "{}:{}",
+        FLUVIO_BENCH_RECORDS_PER_BATCH.0,
+        FLUVIO_BENCH_RECORDS_PER_BATCH.env_or_default()
+    );
+    println!(
+        "{}:{}",
+        FLUVIO_BENCH_RECORD_NUM_BYTES.0,
+        FLUVIO_BENCH_RECORD_NUM_BYTES.env_or_default()
+    );
+    println!(
+        "{}:{}",
+        FLUVIO_BENCH_MAX_BYTES_PER_BATCH.0,
+        FLUVIO_BENCH_MAX_BYTES_PER_BATCH.env_or_default()
+    );
+    let mut group = c.benchmark_group("throughput");
+
+    group.throughput(Throughput::Bytes(
```

Comment:
Throughput is calculated based on the number of bytes per iteration as specified here.